kitsune/questions/models.py

```diff
@@ -122,6 +122,7 @@ class Question(AAQBase):
     )
     is_locked = models.BooleanField(default=False)
     is_archived = models.BooleanField(default=False, null=True)
+    is_failed_deflection = models.BooleanField(default=False)
```

Comment:
We could avoid this by having something like a `@property` and store a key about failed deflections in the question metadata. We can use the add_metadata method for this.

Reply:
We should also make it clear somehow (in the variable name, GA4 events, something else?) that this is related to the first submissions of the question and it's measured against the knowledge base. Question threads also deflect. A thread that has this attribute set to True, can potentially deflect many users from submitting more questions.